### PR TITLE
Solutions-block, carousal-block, footer-changes

### PR DESCRIPTION
**Pull request by varunchinna222**

### CODE REVIEW

File: src/app/footer/footer.component.html [hunk 1]
```diff
@@ -17,19 +17,19 @@
                         <h4>Quick LInk</h4>
                         <ul class="navbar-nav">
                             <li class="nav-item active">
-                                <fa-icon class="link-icon" [icon]="['fas', 'angle-right']" size="xs"></fa-icon>
+                                <!-- <fa-icon class="link-icon" [icon]="['fas', 'angle-right']" size="xs"></fa-icon> -->
```
Comment (ajayreddy002) [inline src/app/footer/footer.component.html]: Remove commented code

File: src/app/home/home.component.scss [hunk 2]
```diff
@@ -118,4 +118,103 @@
...
+    line-height: 210px;
+    text-align: center;
+    color: rgba(0, 0, 0, 0);
+    background-repeat: no-repeat;
+    background-size: cover;
+    -webkit-background-clip: text;
+    background-clip: text;
+    background-position: center center;
+    background-attachment: fixed;
+    background-image: url("https://wordpress.zozothemes.com/briddge/wp-content/uploads/sites/2/2022/11/cta-back-1-1.webp");
```
Comment (ajayreddy002) [inline src/app/home/home.component.scss]: Use image from local assets. Don't use other site urls

File: src/app/home/home.component.scss [hunk 3]
```diff
@@ -118,4 +118,103 @@
...
+    font-weight: 700;
+    line-height: 50px;
+}
+
+.me-3 {
+    font-size: 25px;
+}
+
+.carousal-item {
+    background-image: url("https://wordpress.zozothemes.com/briddge/wp-content/uploads/sites/2/2023/07/Scribble-8-1-1.png");
```
Comment (ajayreddy002) [inline src/app/home/home.component.scss]: Same as above

File: src/app/home/home.component.scss [hunk 4]
```diff
@@ -118,4 +118,103 @@
...
+.carousel-control-prev-icon,
+.carousel-control-next-icon {
+    background-size: 20px;
+    background-color: #0061ea !important;
+}
+
+.carousel-indicators {
+    bottom: -70px !important;
+
+    button {
```
Comment (ajayreddy002) [inline src/app/home/home.component.scss:214]: Why you are using important for most of the classes

**Review by ajayreddy002 — CHANGES_REQUESTED** [review]

File: src/app/footer/footer.component.html (hunk 1 above)
Comment (varunchinna222) [reply src/app/footer/footer.component.html]: Removed commented code

File: src/app/home/home.component.scss (hunk 2 above)
Comment (varunchinna222) [reply src/app/home/home.component.scss]: Updated local images

File: src/app/home/home.component.scss (hunk 3 above)
Comment (varunchinna222) [reply src/app/home/home.component.scss]: updated local images

File: src/app/home/home.component.scss (hunk 4 above)
Comment (varunchinna222) [reply src/app/home/home.component.scss:214]: Removed unwanted important for classes

**Review by ajayreddy002 — APPROVED** [review]

**Pull request merged**